packages/cli/generation/local-generation/local-workspace-runner/src/runLocalGenerationForWorkspace.ts

```diff
@@ -22,13 +23,15 @@ export async function runLocalGenerationForWorkspace({
     keepDocker,
     context,
     irVersionOverride,
+    outputModeForSeedConfig,
 }: {
     organization: string;
     workspace: FernWorkspace;
     generatorGroup: GeneratorGroup;
     keepDocker: boolean;
     context: TaskContext;
     irVersionOverride: string | undefined;
+    outputModeForSeedConfig: FernGeneratorExec.OutputMode | undefined;
```

Comment:
so i think we should expose 2 methods here: 
`runLocalGenerationForWorkspace`
and 
`runLocalGenerationForSeed`

Reply:
only the second method should take an `irVersionOverride`